docs/products/opensearch/howto/upgrade-to-opensearch.rst

```diff
@@ -0,0 +1,45 @@
+Upgrade to OpenSearch
+=====================
+
+For current Aiven for Elasticsearch customers, upgrade is recommended as soon as possible so that we can continue to support your database requirements. Depending on your needs, the various options for testing and upgrading the service are detailed here. The upgrade does not require downtime, and any Kibana service alongside Elasticsearch will be upgraded to OpenSearch Dashboards at the same time.
+
+.. warning::
+    We strongly recommend that you avoid upgrading any library dependencies as part of this upgrade process. Keep your current dependencies at their current versions.
+
+There are three approaches that you can use to upgrade an existing service:
+
+* Create a `fork :doc:/docs/platform/concepts/database-forking` of Aiven for OpenSearch from your existing Aiven for Elasticsearch service. This is a good way to get a fast preview of how Aiven for OpenSearch will work for you.
```

Comment:
Link is not rendering properly, should be >> :doc:`fork </docs/platform/concepts/database-forking>`